cmd/snap-update-ns/mount-entry.h

```diff
@@ -30,6 +30,7 @@
 struct sc_mount_entry {
 	struct mntent entry;
 	struct sc_mount_entry *next;
+	unsigned flag;		// general-purpose flag, not compared
```

Comment:
If this is a general purpose-purpose flag (with a prospect of holding more states in the future), then I'd add a const (#define) for the possible states (just one state for now) and make sure if (d->flag) checks for the exact state. Alternatively, if we don't expect multiple states, then I'd rename "flag" to clearly indicate what is it about.